src/util.ts

```diff
@@ -1,18 +1,18 @@
 import { Api } from 'chessground/api';
 import { Color, Key } from 'chessground/types';
+import {ChessInstance, SQUARES} from 'chess.js';
 
-export function toDests(chess: any): Map<Key, Key[]> {
+export function toDests(chess: ChessInstance): Map<Key, Key[]> {
   const dests = new Map();
-  chess.SQUARES.forEach(s => {
```

Comment:
Since v13, `SQUARES` is now exported directly from chess.js rather than from a chess instance.